journeyman/webdev/svg.md

```diff
@@ -0,0 +1,25 @@
+# SVG (Scalable Vector Graphics)
+
+## Intro
+
+SVG is a Scalable Vector Graphics, which basically means it is a language used for to create two-dimensional vector graphics. These graphics can be used to create complex or simple graphics that can be animated in a variety of ways.
+
+## Suggested Learning
+
+- Give yourself a basic feel for SVG on [W3Schools.](https://www.w3schools.com/graphics/svg_intro.asp)
+- Learn other, more common ways to use SVG at [css-tricks.](https://css-tricks.com/using-svg/)
+- Learn how to use the `<path>` element [here.](https://developer.mozilla.org/en-US/docs/Web/SVG/Tutorial/Paths)
+
+## Requirements
+- Make a project that demonstrats the points below:
```

Comment:
Technically, yes. But no. I meant HTML. HTML and SVG are both subsets of XML and embedding SVG in HTML documents is a supported practice. The project needs to be an HTML document since you can't use `<img />` tags and the `background-image` CSS property in non-HTML XML documents.